tests/fixtures/integration/actions/lint/test_direct_interactive_noee.py/test/0.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "test[0-ansible-navigator lint /tests/fixtures/integration/actions/lint --ee False --ll debug --mode interactive-lint interactive without ee]",
+    "index": 0,
+    "comment": "lint interactive without ee",
+    "additional_information": {
+        "present": [
+            "Issues were found while applying the settings"
+        ],
+        "absent": [],
+        "compared_fixture": false
+    },
+    "output": [
+        "Warning: Issues were found while applying the settings.",
```

Comment:
These tests should pass and have output similar to what the tests w/o the EE get.... This looks like ansible-lint with an EE doesn't work at all...

Reply:
yes, right now ansible-lint does not work without execution environment enabled. Okay, so I'll add lint as a requirement first and then update these tests to reflect the same output.